vowpalwabbit/global_data.h

```diff
@@ -264,6 +264,8 @@ struct workspace
 
   vw_logger logger;
   bool audit;     // should I print lots of debugging information?
+  std::shared_ptr<std::vector<char>> _audit_buffer;

```

Comment:
[Minor] Consider keeping the naming convention for members consistent within objects.

Reply:
Happy to fix